src/Sylius/Bundle/TaxonomyBundle/Resources/config/services.xml

```diff
@@ -32,6 +33,10 @@
             <argument type="service" id="sylius.repository.taxon" />
             <tag name="form.type" alias="sylius_taxon_selection" />
         </service>
+
+        <service id="sylius.taxonomy_factory" class="%sylius.taxonomy_factory.class%">
```

Comment:
Let's use `sylius.factory.taxonomy` to be consistent with Resource `sylius.(repository | model | form).resource_name` naming convention.